### PR TITLE
Set1Task6 and Set1Task7

### LINKED ISSUE #8: Set1Task6: Add decision logic to your code using the if-elseif-else statement

### LINKED ISSUE #9: Set1Task7: Store and iterate through sequences of data using Arrays and the foreach statement

### PR DESCRIPTION
**Pull request by juturul**
Closes #8 
Closes #9

### CODE REVIEW

File: Lavanya.C.Sharp/Set1Task2/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (ShriHumrudha) [inline Lavanya.C.Sharp/Set1Task2/Properties/launchSettings.json]: Why do we need this file?

File: Lavanya.C.Sharp/Set1Task6/Program.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
+﻿
```
Comment (ShriHumrudha) [inline Lavanya.C.Sharp/Set1Task6/Program.cs:1]: Run the app and share the console screenshot in the comments.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by juturul** [comment]
Set1Task6 output:
image

Set1Task7 Output:
image

File: Lavanya.C.Sharp/Set1Task2/Properties/launchSettings.json (hunk 1 above)
Comment (juturul) [reply Lavanya.C.Sharp/Set1Task2/Properties/launchSettings.json]: Removed it.

File: Lavanya.C.Sharp/Set1Task6/Program.cs (hunk 2 above)
Comment (juturul) [reply Lavanya.C.Sharp/Set1Task6/Program.cs:1]: I posted output for both Set1Task6 and Set1Task7 in the comments.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Points are displayed as 25 but how it is 2.5 for a tie.

File: Lavanya.C.Sharp/Set1Task7/Program.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
+﻿
+using System.Collections.Generic;
+
+List<int> playerOne = new List<int> { 1, 5, 10, 25, 50 };
```
Comment (ShriHumrudha) [inline Lavanya.C.Sharp/Set1Task7/Program.cs]: Assign random values for both the players.

**Comment by juturul** [comment]
> Points are displayed as 25 but how it is 2.5 for a tie.

25 is presented coin value, not points

**Review by ShriHumrudha — APPROVED** [review]

File: Lavanya.C.Sharp/Set1Task7/Program.cs (hunk 3 above)
Comment (juturul) [reply Lavanya.C.Sharp/Set1Task7/Program.cs]: fixed

**Pull request merged**